tests/bundle/test_sparql.py

```diff
@@ -0,0 +1,23 @@
+from onto_tool import onto_tool
+import csv
+
+
+def lists_equal(list_one, list_two):
+    return len(list_one) == len(list_two) and sorted(list_one) == sorted(list_two)
+
+
+def test_sparql_queries():
+    onto_tool.main([
+        'bundle', 'tests/bundle/sparql.yaml'
+    ])
+    with open('tests/bundle/sparql.csv') as csvfile:
```

Comment:
The way I read this it is opening an existing file rather than creating a new one, but I could be wrong. I don't see a sparql.csv file in the directory.

Reply:
It opens a file that is created by the previous command. If you look at `sparql.yaml`, it executes two actions - one performs a `CONSTRUCT` query to create `sparql.ttl`, the other runs a `SELECT` query on that output to create `sparql.csv`, then the test verifies the contents of that file.